packages/runtime/test/fixtures/stateful/assets/close-clear-ssc.teal

```diff
@@ -0,0 +1,52 @@
+// check if application is being created
+// This ssc only allows creator to close out
+int 0
+txn ApplicationID
+==
+
+bz not_creation
+
+// save the creator address to the global state
+byte "Creator"
+txn Sender
+app_global_put
+int 1
+return
+
+not_creation:
+int CloseOut
+txn OnCompletion
+==
+int ClearState
+txn OnCompletion
+==
+||
+
+// finish if not closeOut/Clear call
+bz finished
+
+// check if creator is closing out
+byte "Creator"
+app_global_get
+txn Sender
+==
+bz failed
+
+// update global state before closing out
+byte "global-key"
+byte "global-val"
+app_global_put
+
+// update txn sender's local state before closing out
+int 0 //sender
+byte "local-key"
+byte "local-val"
+app_local_put
```

Comment:
we are closing out, so i think this will be removed. i don't think we need to update local state

Reply:
Yes, correct. I actually wanted to cover these cases. In this contract we are updating the global state, and local state as well. But since this is a closeOut/ClearState: a) globalState should be updated (and it should NOT be removed) b) local state should not be updated as it is removed

I don't see any benefit of not doing it. Let me know if you still think we should remove, i'll remove it then

Reply:
i think according we are updating local state here. so that code is unused as we are deleting local state, for global state yes we can update, but for local, i think it is confusing as why we are doing it local state will be deleted.

Reply:
ok, removing local state lines